valid-palindrome/kimyoung.js

```diff
@@ -0,0 +1,23 @@
+var isPalindrome = function (s) {
+    let left = 0, right = s.length - 1;
+    while (left < right) {
+        while (left < right && !isAlphaNumeric(s[left])) { // increment the left index until it's an alphanumeric character
+            left++;
+        }
+        while (left < right && !isAlphaNumeric(s[right])) { // decrement the right index until it's an alphanumeric character
+            right--;
+        }
+        if (s[left++].toLowerCase() !== s[right--].toLowerCase()) return false; // compare the two string values, if different return false;
+    }
+    return true;
+};
+
+function isAlphaNumeric(char) { // use ASCII code to findout if char is alphanumeric or not
+    const asciiCode = char.charCodeAt(0);
+    return (asciiCode >= 65 && asciiCode <= 90) ||
+        (asciiCode >= 97 && asciiCode <= 122) ||
+        (asciiCode >= 48 && asciiCode <= 57)
```

Comment:
ASCII 코드를 다 외우고 있지 않아서, 코드가 좀 읽기가 어렵네요... 😵‍💫
그런데 굳이 ASCII 코드로 변환을 해야할까요? 코드 가독성을 위해 그냥 문자열로 비교하면 어떨까요?

```suggestion
    return (char >= "A" && char <= "Z") ||
        (char >= "a" && char <= "z") ||
        (char >= "0" && char <= "9")
```